HPCC-13703 DOCS:documentation for systemd script

g-pan opened the pull request:
FIX HPCC-13703 DOCS:documentation for systemd script
Update Installing and Running doc. for systemd, script.

Signed-off-by: G Panagiotatos greg.panagiotatos@lexisnexis.com
@JamesDeFabia please review
@xwang2713 please review

g-pan committed:
HPCC-13703 DOCS:documentation for systemd script

FIX HPCC-13703 DOCS:documentation for systemd script
Update Installing and Running doc. for systemd, script.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

g-pan committed:
HPCC-13703 DOCS:documentation for systemd script

FIX HPCC-13703 DOCS:documentation for systemd script
Update Installing and Running doc. for systemd, script.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest 
Sha: 5c61a6a5cc34e9c4921fc8c06ed2d5387fe403c7
Build: success
Install hpccsystems-platform-community_6.0.0-rc4.el6.x86_64.rpm
HPCC Start: OK
HPCC Stop: OK
HPCC Uninstall: OK

JamesDeFabia commented on docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml:
```diff
@@ -2722,14 +2717,122 @@ init_start_thor--></property></para>
+        services in this way provides status in the same manner as invoking
+        them with a service command does for other HPCC processes.</para>
+
+        <para>Currently, the hpcc-init system service will only support
```
Remove the word Currently

JamesDeFabia commented on docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml:
```diff
@@ -2722,14 +2717,122 @@ init_start_thor--></property></para>
+        under /lib/systemd/system.</para>
+
+        <para>The first set does not have any output to STDOUT/STDERROR,
+        instead it logs to /var/log/syslog. Users can enter: "journalctl -u
```
Put the cli commands inside a  <programlisting>

JamesDeFabia commented on docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml:
```diff
@@ -2722,14 +2717,122 @@ init_start_thor--></property></para>
+        &lt;service&gt; -f" to see the output. The second set has the same
+        output to STDOUT/STDERROR as directly invoking
+        /etc/init.d/&lt;hpcc-init|dafilesrv&gt;</para>
+
```
You can either use... instead of User can either use

JamesDeFabia commented on docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml:
```diff
@@ -2722,14 +2717,122 @@ init_start_thor--></property></para>
+        <para>HPCC services started through systemd will be listed as active
+        in systemd. They can be listed as "sudo systemctl list-units
+        [PATTERN...]". To remove them from the systemd active service list,
+        the user must run the stop service from the "service" or "systemctl"
```
You instead of the user

JamesDeFabia commented on docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml:
```diff
@@ -2722,14 +2717,122 @@ init_start_thor--></property></para>
+            </orderedlist></para>
+
+          <para>To start/stop HPCC with hpcc-init with STDOUT/STDERR output.
+          Get current tty by run "tty". For example, tty -&gt; /dev/pts/1.
```
did you mean by running

g-pan committed:
HPCC-13703 DOCS:documentation for systemd script

FIX HPCC-13703 DOCS:documentation for systemd script
Update Installing and Running doc. for systemd, script.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest 
Sha: fe6427098d43a3bc17ead0fc4db4010ec4732752
Build: success
Install hpccsystems-platform-community_6.0.0-rc4.el6.x86_64.rpm
HPCC Start: OK
HPCC Stop: OK
HPCC Uninstall: OK

JamesDeFabia commented:
@richardkchapman  Looks Good to merge
